LIS/lis-test
# Add vssd disable enable vssd

Add one case about backup when vss is disabled and enabled in the "integration service", if VSS is disabled, it should do offline backup, otherwise do online backup.

Note: Refer to bug https://bugzilla.redhat.com/show_bug.cgi?id=1074407,  after kernel version kernel-3.10.0-383.el7, the check or un-check VSS could take effect without vm reboot.

Case Description: 
Check  'Backup(volume snapshot)' after un-check 'Backup(volume snapshot)' integration services from Hyper-V Manager and do backup after un-check and check

Test Steps:
1. Right click your VM in Hyper-V Manager, select ""Settings"", change to
 ""Integration Services"" and un-check ""Backup (volume snapshot)"""
2. Backup VM from server host
3. Change to ""Integration Services"" from Hyper-V Manager settings and check
 ""Backup (volume snapshot)"""
4. Repeat step 2
Note: for RHEL6, refer to not fix bug https://bugzilla.redhat.com/show_bug.cgi?id=1121888 , all the integration servers after check/uncheck, need to reboot VM to make the option take effect."

Expected Result:
After step 2, The online backup should be failed, and host should display offline backup log:
 "Copying files for "$VM_name"(Offline)... xx% done. "
 Check VM backup log "Caption: Offline\$VM_name"
 After step 4, The backup should be success,Host should display:
 "Copying files for "$VM_name"(Online)... xx% done. "
 Check VM backup log "Caption: Online\$VM_name"
